tools/istio-docker.mk

```diff
@@ -195,7 +198,7 @@ SECURITY_DOCKER:=docker.citadel docker.citadel-test docker.node-agent docker.nod
 $(SECURITY_DOCKER): security/docker/Dockerfile$$(suffix $$@) | $(ISTIO_DOCKER)
 	$(DOCKER_RULE)
 
-DOCKER_TARGETS:=docker.pilot docker.proxy_debug docker.proxytproxy docker.proxyv2 docker.app docker.test_policybackend $(PILOT_DOCKER) $(SERVICEGRAPH_DOCKER) $(MIXER_DOCKER) $(SECURITY_DOCKER) $(GALLEY_DOCKER)
+DOCKER_TARGETS:=docker.pilot docker.proxy_debug docker.proxytproxy docker.proxyv2 docker.app docker.test_policybackend $(PILOT_DOCKER) $(SERVICEGRAPH_DOCKER) $(MIXER_DOCKER) $(SECURITY_DOCKER) $(GALLEY_DOCKER) docker.kubectl
```

Comment:
I'm not sure this image must be built on each make invocation. It is pretty slow and doesn't change often. 
For example we don't build the base image every time, or the circle ci. 

You can use a separate target.

Reply:
sure, I already have a seperate target (docker.kubectl).  Should I just delete this target from the build-all-the-time target list?  If so wfm, although someone will have to build and push the kubectl target manually (much as istio_nightllydebug is handled).  Is there a better way?